registrations/management/commands/delete_expired_seatreservations.py

```diff
@@ -0,0 +1,26 @@
+from datetime import timedelta
+
+from django.core.management import BaseCommand
+from django.db.models import DateTimeField, ExpressionWrapper, F
+from django.utils.timezone import localtime
+
+from registrations.models import SeatReservationCode
+from registrations.utils import code_validity_duration
+
+
+class Command(BaseCommand):
```

Comment:
This is fine as is and might be more of a philosophical debate, but I can't help myself thinking this is a different feature than the filtering feature 🧐 So in context of this being included in `feat: allow filter events by...` commit I can understand that this is included and related in that feature but think still this would be in its own feature complementing the filtering one (also the one regarding the cron running this). 💭 
Same thinking applies also to the calculations for the capacities ☕